kyuubi-ctl/src/test/scala/org/apache/kyuubi/ctl/ServiceControlCliArgumentsSuite.scala

```diff
@@ -67,6 +67,11 @@ class ServiceControlCliArgumentsSuite extends KyuubiFunSuite {
       }
       thread.start()
       thread.join()
+      // scalastyle:off println
+      println(logAppender.loggingEvents)
+      println("------")
+      println(searchString)
+      // scalastyle:on println
```

Comment:
Shall we remove those debug `println`?

Reply:
Sorry , I added it when I improved ut locally and forgot to remove it.